plugin/error_vis.py

```diff
@@ -156,6 +185,11 @@ def remove_region(self, view_id, row):
         del current_error_dict[row]
 
     @staticmethod
+    def _on_phantom_navigate(self):
+        """Close all phantoms in active view."""
+        sublime.active_window().active_view().erase_phantoms("compile_errors")
```

Comment:
I guess it would be better to have a tag that starts with "ECC" just to make sure we don't interfere with other plugins. Also, `"compile_errors"` should be stored in some static variable in `CompileErrors` class.